rust/agama-cli/src/questions.rs

```diff
@@ -5,9 +5,19 @@ use clap::{Args, Subcommand, ValueEnum};
 
 #[derive(Subcommand, Debug)]
 pub enum QuestionsCommands {
-    /// Set mode for answering questions.
+    /// Sets the mode for answering questions.
+    ///
+    /// It allows to decide if questions will be interactive or
+    /// if it should not block installation.
     Mode(ModesArgs),
+    /// Loads predefined answers to questions.
+    ///
+    /// It allows to predefine answers for certain questions to skip
+    /// them in interactive mode or change answer in automatic mode.
+    ///
+    /// For more details and examples see official Agama documentation.
```

Comment:
I'd like to have a more helpful reference here

Reply:
https://github.com/openSUSE/agama/blob/master/doc/questions.md or anything else?

Reply:
That is what I am not sure if it will be documented on official suse side or not..and if it will be product specific or not.